### ISSUE TITLE
Getting Error 400 after external update [INVALID_ARGUMENT on ColorTemp lights]

### ISSUE THREAD

**Issue by Aragur** [mikejac/node-red-contrib-google-smarthome#65]
```
HttpActions:reportState(): error; {"error":{"code":400,"message":"Request contains an invalid argument.","status":"INVALID_ARGUMENT"}}
```

**Comment by Caprico85** [comment]
INVALID_ARGUMENT errors happen, when an invalid device state is reported to Google. For example a window that is opened 120% (only 0-100% allowed). These errors are usually not critical. The worst thing that can happen is Google saying/showing an invalid value. Should be fixed nonetheless.

To investigate this further can you please increase Node-Red's log level to debug (set "level" to "debug" in settings.js) and post the last few lines from Node-Red's log output before the error is logged. There should be a line like "HttpActions:reportState(): states = ...", followed by a JSON object. I especially need this JSON object.

**Comment by Aragur** [comment]
https://pastebin.com/HXSUcs81

**Comment by Caprico85** [comment]
Somehow the most interesting lines were not printed. Let's please try something else. I released v0.0.48, which now shows a more verbose error message on the debug panel. Please copy this error message. I especially need the "states" object in there.

![image](https://user-images.githubusercontent.com/2081806/100732848-a08f7300-33cd-11eb-828a-c3e7f3509c3d.png)

**Comment by Aragur** [comment]
![chrome_2020-12-01_20-47-31](https://user-images.githubusercontent.com/11004008/100789249-7b732280-3416-11eb-8deb-923130b5b805.png)
PS: nice to see another German Dev here :) Got you by "komischerwert"

**Comment by Caprico85** [comment]
Thank you for the screenshot. I'm finally able to reproduce this. The node creates a wrong state report to Google. Only `color.temperatureK` is allowed by Google, not the `temperature` value outside of color. I'll see what I can can do.

PS: Erwischt. Grüße aus dem schönen Sachsen 😄

**Comment by Caprico85** [comment]
Should be fixed in v0.0.49.
